### PR TITLE
interrogator lamp

### PR DESCRIPTION
**Pull request by FoxHorn**
Ultra-bright lamp for the bad cop. Can blind like a flash. For significant pressure on the suspect.
image3

## About the PR

**Media**

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added prototype interrogator lamp for admin entity spawn panel. Perhaps the mapmakers will replace the lamps in the interrogation rooms with this? I said where's the detonator?!?!

### CODE REVIEW

**Comment by Dynexust** [comment]
There's no pictures?

**Comment by FoxHorn** [comment]
image1
image2

**Comment by FoxHorn** [comment]
> There's no pictures?

Added

**Comment by OctoRocket** [comment]
40 charges?

**Comment by FoxHorn** [comment]
> 40 charges?

I'm thinking of doing a slow charge regen and reducing the charges to a small amount in the future.

File: Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml [hunk 1]
```diff
@@ -104,6 +104,52 @@
...
+    charges: 40
+  - type: MeleeWeapon
+    damage:
+      types:
+        Blunt: 0 # melee weapon to allow flashing individual targets
+  - type: GenericVisualizer
+    visuals:
+      enum.FlashVisuals.Burnt:
+        enum.FlashVisuals.BaseLayer:
+          True: {state: burnt}
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml:147]: Your RSI doesn't have `burnt` state

File: Resources/Textures/Objects/Misc/Lights/lampint.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    {
+        "name": "flashing",
+      	"delays": [
+          [
+            0.1,
+            0.1,
+            0.3
+          ]
+      	]
```
Comment (ficcialfaint) [inline Resources/Textures/Objects/Misc/Lights/lampint.rsi/meta.json:24]: Fix the tabulations

File: Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml (hunk 1 above)
Comment (FoxHorn) [reply Resources/Prototypes/Entities/Objects/Misc/fluff_lights.yml:154]: Thanks for noticing, I fixed it

File: Resources/Textures/Objects/Misc/Lights/lampint.rsi/meta.json (hunk 2 above)
Comment (FoxHorn) [reply Resources/Textures/Objects/Misc/Lights/lampint.rsi/meta.json]: Fixed

**Comment by Arteben** [comment]
I'm really confused by the number of charges. This is not the case for the lamp spotlight of the salvagers, but it is for the table lamp. This thing is really unbalanced.

**Comment by FoxHorn** [comment]
> I'm really confused by the number of charges. This is not the case for the lamp spotlight of the salvagers, but it is for the table lamp. This thing is really unbalanced.

she now has 3 charges with slow recovery

**Pull request merged**